controllers/controllers/services/bindings/controller.go

```diff
@@ -145,6 +144,10 @@ func (r *Reconciler) ReconcileResource(ctx context.Context, cfServiceBinding *ko
 		return res, err
 	}
 
+	if cfServiceBinding.Spec.Type == korifiv1alpha1.CFServiceBindingTypeKey {
+		return ctrl.Result{}, nil
```

Comment:
There is no controller test that asserts that no servicebindingio bindings are created to bindings of type `key`. Please add one